nautobot_golden_config/migrations/0010_convert_sotagg_queries.py

```diff
@@ -0,0 +1,78 @@
+from datetime import date
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def save_existing_sotagg_queries(apps, schema_editor):
+    """
+    Save to the temp field the current SoTAgg Query strings.
+    """
+    GoldenConfigSetting = apps.get_model("nautobot_golden_config", "GoldenConfigSetting")
+
+    for gc_setting_obj in GoldenConfigSetting.objects.all():
+        if gc_setting_obj.sot_agg_query:
+            gc_setting_obj.sot_agg_query_tmp = gc_setting_obj.sot_agg_query
+            gc_setting_obj.save()
+
+
+def create_and_link_gql_queries(apps, schema_editor):
+    """
+    Create Saved GraphQL Query objects and link them to SoTAgg Query field.
+    """
+    GoldenConfigSetting = apps.get_model("nautobot_golden_config", "GoldenConfigSetting")
+    GraphQLQuery = apps.get_model("extras", "GraphQLQuery")
+
+    today = str(date.today())
+
+    for gc_setting_obj in GoldenConfigSetting.objects.all():
+        if gc_setting_obj.sot_agg_query_tmp:
+            sotagg_name = gc_setting_obj.name
+            sotagg_query = gc_setting_obj.sot_agg_query_tmp
+            gqlsq_name = f"GC {sotagg_name} - {today}"
+
+            gqlsq_obj = GraphQLQuery()
+            gqlsq_obj.name = gqlsq_name
+            gqlsq_obj.query = sotagg_query
+            gqlsq_obj.variables = {"device_id": ""}
+            gqlsq_obj.save()
+
+            gc_setting_obj.sot_agg_query = gqlsq_obj
+            gc_setting_obj.save()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("extras", "0018_joblog_data_migration"),
+        ("nautobot_golden_config", "0009_multiple_gc_settings"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="goldenconfigsetting",
+            name="sot_agg_query_tmp",
+            field=models.TextField(blank=True),
+        ),
+        migrations.RunPython(save_existing_sotagg_queries),
```

Comment:
Please check #222 regarding `RunPython` within migrations.

Reply:
I split the migrations into individual steps following pattern established in #223